src/structures/point.py

```diff
@@ -76,11 +76,14 @@ def __lt__(self, other: object) -> bool:
         assert isinstance(other, Point), "Must compare with another point"
         return self._pos < other._pos
 
+    def __add__(self, other: object) -> 'Point':
```

Comment:
Adapt this for other operations, see `Colour`, it already does this. You could probably just copy paste what colour does and use generic `zip` instead of the manual unpacking I did for `Colour`.

Reply:
Implemented this in my branch, revert changes to this.